apps/communications/contacts/test/marionette/lib/contacts.js

```diff
@@ -415,6 +415,22 @@ Contacts.prototype = {
     return dialer;
   },
 
+  addToContactAndReturnToDialer: function(contactName) {
+    this.tapContact(contactName);
+    this.client.helper.waitForElement(Contacts.Selectors.formSave).click();
```

Comment:
`createNewContactAndReturnToDialer()` implements a part of this behavior. The 5 lines below can be factorized in a function like:
```js
_saveAndReturnToDialer: function() {}
```